openfe/protocols/openmm_utils/omm_settings.py

```diff
@@ -371,3 +370,49 @@ def must_be_positive(cls, v):
                       "intervals must be positive")
             raise ValueError(errmsg)
         return v
+
+
+class SimulationSettingsMD(SimulationSettings):
+    """
+    Settings for simulation control for plain MD simulations, including
+    writing outputs
+    """
+    class Config:
+        arbitrary_types_allowed = True
+
+    equilibration_length_nvt: unit.Quantity
+    """
+    Length of the equilibration phase in the NVT ensemble in units of time. 
+    The total number of steps from this equilibration length
+    (i.e. ``equilibration_length_nvt`` / :class:`IntegratorSettings.timestep`)
+    must be a multiple of the value defined for
+    :class:`IntegratorSettings.n_steps`.
+    """
+    # reporter settings
+    output_filename = 'simulation.xtc'
+    """Path to the storage file for analysis. Default 'simulation.xtc'."""
+    trajectory_interval = 5000 * unit.timestep
```

Comment:
```suggestion
    trajectory_write_interval = 5000 * unit.timestep
```
(please see this as only a suggestion - there are probably better names)

Reply:
changed this!